framework/build.gradle

```diff
@@ -239,13 +239,11 @@ createScript(project, 'org.tron.program.SolidityNode', 'SolidityNode')
 createScript(project, 'org.tron.program.FullNode', 'FullNode')
 createScript(project, 'org.tron.program.KeystoreFactory', 'KeystoreFactory')
 createScript(project, 'org.tron.program.DBConvert', 'DBConvert')
-createScript(project, 'org.tron.tool.litefullnode.LiteFullNodeTool', 'LiteFullNodeTool')
```

Comment:
Some comments should be updated such as this :https://github.com/tronprotocol/java-tron/blob/4159b3a7c70fef5b2fe91ea34ff9704e869fe05a/plugins/src/main/java/org/tron/plugins/utils/FileUtils.java#L131